docs/source/tutorials/qaoa.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.    # for other CPUs, please use tf.keras.optimizers.Adam
</pre><p>why the suggestion for M mac?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tencent-quantum-lab/tensorcircuit/pull/148/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
because I have an M series Mac and I always get wired worning messages with `tf.keras.optimizers.Adam`
but now I found a method to hide these messages.